packages/components/empty-state/src/MissingContent/MissingContent.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: 'MissingContent'
+type: 'component'
+slug: /empty-state/missingcontent/
+section: 'utils'
```

Comment:
We may remove the section, since we don't have other components on the empty state section at the moment